core/tracepoint/disable_spec.rb

```diff
@@ -0,0 +1,20 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#disable' do
+    context 'deactivates the trace' do
```

Comment:
Not really a context. We need a test that shows that a trace doesn't call the block after it's disabled...
Can probably be shared with `enable`

Reply:
I didn't get - how can it be shared with `enable`.